reference/halyard/high-availability.md

```diff
@@ -186,4 +186,8 @@ With all services enabled for high availability, the new architecture looks like
 
  {% include mermaid %}
 
+## Recoverability
 
+If you've [configured an external Redis](/setup/productionize/caching/externalize-redis/), Spinnaker can recover from failure events. Although it's possible, it is not recommended to use spot instances on AWS or preemtible nodes to lower cost in production, as outages in your continuous deployment tool will likely cost more than any savings.
```

Comment:
@milesmatthias 
Rather than follow the statement that it's possible to recover with a statement about what's not recommended, could you make the 2nd sentence be a very brief summary of how to recover? And if it's a multi-step process that needs to be documented explicitly, you could elaborate after this paragraph.

And move the not-recommended statement to a separate paragraph below what *is* recommended?

Reply:
Done @dorbin